loopFibonacci.c

```diff
@@ -0,0 +1,28 @@
+// Justin Yu   I'm getting infinite loop but I have no idea why //
+
+#include <stdio.h>
+
+int main() {
+
+int first, second, next, x, i;
+first = 0;
+second = 1;
+
+printf("Enter the number of terms\n");
+scanf("%d", &x);
+
+printf("The terms are %d\n", x);
+
+for( i = 0; i < x; i++) {
+if (i <= 1) 
+	i = next;
+else {
```

Comment:
not sure why you chose to do it this way - explain and i can give you some insight as to what went wrong